### ISSUE TITLE
MS11 - Overall description of the testbed facilities at LAL 

### ISSUE THREAD

**Issue by cblanchet** [cyclone-project/cyclone-tracking#89]
Related to the task #81 
Provide a overall description of the hardware (CPU, RAM, storage, network)

**Comment by lodygens** [comment]
This is available in onedrive/cyclone/wp7-testbed/m7.1/cyclone_m7.1-v1.1-lal

**Comment by lodygens** [comment]
There is also a real time kind of monitoring page here : https://cloud.lal.stratuslab.eu/load.txt

**Comment by cblanchet** [comment]
Could you summarize it in one or two paragraphs. We may not need such a detailed description for the MS11.

**Comment by lodygens** [comment]
Is there any document in onedrive?

By the way here it is:
Stratuslab@LAL:
- "one proxy" : the authentication server (https://cloud.lal.stratuslab.eu)
- "pdisk": the virtual disk service management (https://pdisk.lal.stratuslab.eu)
- "MarketPlace": the image management (http://marketplace.stratuslab.eu/)

The deployment where virtual machines run, is made of:
- 15 servers 
  * 32 and 24 cores
  * 35 and 64Gb of RAM

**Comment by Josyab** [comment]
@lodygens: I believe this is for the MS11 document. It is required  a section in which you describe and proide the details of the LAL infrastructure: the available resources (what you just detailed), where are tehy located, how are they connected, how are they going to be part of the CYCLONE cloud federated infrastructure. It has to be us much detailed as possible, and in a "nice-shape" to be part of the document. All this information (with some explicative figures if possible) should be included in section 4.1 of the document.

**Comment by lodygens** [comment]
ok thanks
I have copy/pased to MS11
:)

**Comment by cblanchet** [comment]
CNRS provided the required contribution
